### PR TITLE
feat(HACBS-1213): add CycloneDX CLI

### PR DESCRIPTION
**Pull request by hongweiliu17**
Signed-off-by: Hongwei Liu hongliu@redhat.com

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,6 +3,7 @@ FROM registry.access.redhat.com/ubi8/ubi
 
 ARG conftest_version=0.33.2
 ARG BATS_VERSION=1.6.0
+ARG cyclonedx_version=0.24.0
```
Comment (hongweiliu17) [inline Dockerfile:6]: Actually I wanted to get the newest tag from https://github.com/CycloneDX/cyclonedx-cli/tags and set it to `cyclonedx_version`  so we don't need to update the version but have not found a quick way to do it.

Comment (dirgim) [reply Dockerfile:6]: I think that we should be fine using a fixed version for now, and we can consider making periodic updates to our tooling versions in the future.
About the current version - would it be better to use the newest [v0.24.2](https://github.com/CycloneDX/cyclonedx-cli/releases/tag/v0.24.2) version here?

Comment (hongweiliu17) [reply Dockerfile:6]: Ok, thanks!

**Review by dirgim — APPROVED** [review]
LGTM

**Comment by dirgim** [comment]
Note: the integration tests failure is unrelated to these changes and will be fixed in #32

**Review by jsztuka — APPROVED** [review]
lgtm

**Pull request merged**